engine/events/print_unown.asm

```diff
@@ -74,7 +74,13 @@ _UnownPrinter:
 	jr nz, .pressed_b
 
 	ldh a, [hJoyPressed]
+	vc_patch print_forbid_1
+if DEF(_CRYSTALVC)
```

Comment:
@Rangi42 Would you prefer if `_CRYSTALVC` was renamed to `_CRYSTAL_VC`?

Reply:
It's fine as-is.